### PR TITLE
Benreichert/formatting

### PR DESCRIPTION
**Pull request by fernviridian**

### CODE REVIEW

**Review by pdouglas94 — APPROVED** [review]

**Comment by dnseitz** [comment]
Seems like we should just run the script then merge in the header changes rather than merging in the script

File: altos-rust/altos-core/src/arch/cm0.rs [hunk 1]
```diff
@@ -76,7 +109,8 @@ pub fn in_kernel_mode() -> bool {
   unsafe {
     let stack_mask: usize;
     #[cfg(target_arch="arm")]
-    asm!("mrs $0, CONTROL\n" /* get the stack control mask */
```
Comment (fernviridian) [inline altos-rust/altos-core/src/arch/cm0.rs]: the script removed the \n....grr

File: altos-rust/port/cortex-m0/src/lib.rs [hunk 2]
```diff
@@ -139,16 +168,26 @@ fn init_bss_segment() {
...
-                "movs r0, #0\n", /* store zero for later */
-                "ldr r1, =_sbss\n", /* start of bss in RAM */
-                "ldr r2, =_ebss\n", /* end of bss in RAM */
-                "loop:\n",
-                "cmp r1, r2\n", /* check if we've reached the end of our segment */
-                "bpl b_done\n",
-                "str r0, [r1]\n", /* if not, zero out memory at current location */
-                "adds r1, #4\n",
-                "b loop\n", /* repeat until done */
-                "b_done:\n")
```
Comment (fernviridian) [inline altos-rust/port/cortex-m0/src/lib.rs]: script literally interpreted the new lines, was not expecting that....

File: altos-rust/altos-core/src/init.rs [hunk 3]
```diff
@@ -24,10 +41,13 @@
 ///   let heap_size: usize;
 ///   asm!(
 ///     concat!(
-///       "ldr r0, =_heap_start\n",
```
Comment (fernviridian) [inline altos-rust/altos-core/src/init.rs]: this is not a problem since its a comment

File: altos-rust/altos-core/src/tick.rs [hunk 4]
```diff
@@ -25,4 +42,3 @@ pub fn tick() {
 pub fn get_tick() -> usize {
   SYSTEM_TICKS.load(Ordering::Relaxed)
 }
-
```
Comment (fernviridian) [inline altos-rust/altos-core/src/tick.rs:28]: removed an unnecessary line

**Comment by fernviridian** [comment]
Good for review again. Fixed missing newlines.

**Pull request merged**